docs/source/quickstart/local-subgraphs.mdx

```diff
@@ -19,25 +19,13 @@ First, install the latest version of `@apollo/subgraph` and `graphql-tag` into y
 npm install @apollo/subgraph graphql-tag
 ```
 
-Import the `gql` tag into the file where you create your schema (`index.ts` for our example):
+Import the `gql` tag into the file where you create your schema (`index.ts` for our example), and wrap your schema in that tag, like so:
 
 <MultiCodeBlock>
 
 ```ts title="index.ts"
-import gql from 'graphql-tag';
-```
-
-```js title="index.js"
-import gql from 'graphql-tag';
-```
-
-</MultiCodeBlock>
+import gql from 'graphql-tag'; // preserve-line
```

Comment:
I _think_ you don't need the `// preserve-line` anymore since there's more code to accompany this now that shouldn't disappear in the JS? I might be wrong and might've misunderstood the original problem.

Reply:
Oh yes! We can remove that now :)